Dockerfile

```diff
@@ -184,10 +184,11 @@ RUN pip3 install envdir==1.0.1
 RUN pip3 install awscli==1.18.195
 
 # Install Snakemake and related optional dependencies.
-RUN pip3 install snakemake==5.10.0
+# Pinned to 7.24.1 for stability (2023-03-13)
+RUN pip3 install snakemake==7.24.1
 # Google Cloud Storage package is required for Snakemake to fetch remote files
 # from Google Storage URIs.
-RUN pip3 install google-cloud-storage==2.1.0
+RUN pip3 install google-cloud-storage==2.7.0
```

Comment:
What's the reason for bumping `google-cloud-storage`?

Reply:
Not sure, it's an optional dependency of snakemake but doesn't seem to be pinned. So could also leave as is.